docs/bin/plugin_formatter.py

```diff
@@ -132,6 +133,18 @@ def rst_xline(width, char="="):
     return char * width
 
 
+def to_boolean(value):
+    ''' convert default value to boolean if it intends to be boolean '''
```

Comment:
Could you please fix this to follow PEP 257?

Reply:
Oh sorry, I thought you are talking about the empty line between comment and code but not the docstring.